mmdet/models/roi_heads/dynamic_roi_head.py

```diff
@@ -0,0 +1,139 @@
+import torch
+
+from mmdet.core import bbox2roi
+from mmdet.models.losses import SmoothL1Loss
+from ..builder import HEADS
+from .standard_roi_head import StandardRoIHead
+
+
+@HEADS.register_module()
+class DynamicRoIHead(StandardRoIHead):
+    """RoI head for `Dynamic R-CNN <https://arxiv.org/abs/2004.06002>`_."""
+
+    def __init__(self, **kwargs):
+        super(DynamicRoIHead, self).__init__(**kwargs)
+        self.k_i = self.train_cfg.dynamic_rcnn.k_i
+        self.k_e = self.train_cfg.dynamic_rcnn.k_e
```

Comment:
We may replace `k_i`, `k_e` with some explicit name.

Reply:
How about `topk_iou` and `topk_beta`?

Reply:
Looks good to me.

Reply:
Fixed.